### PR TITLE
Add advisors section mobile

### PR DESCRIPTION
**Pull request by filipebfernandes**

### CODE REVIEW

File: src/components/PartnerCardMobile/index.css [hunk 1]
```diff
@@ -0,0 +1,26 @@
+@import "../../styles/colors.css";
+@import "../../styles/layout.css";
+
+.root {
+  display: flex;
+  padding: $spacing-16;
+
+  background-color: white;
+
+  box-shadow: 8px 8px 28px rgba(0, 0, 0, 0.08);
```
Comment (jfranciscosousa) [inline src/components/PartnerCardMobile/index.css]: Use the new color function. color(black alpha(8%))

File: src/assets/images/in-darkBlue.svg [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (jfranciscosousa) [inline src/assets/images/in-darkBlue.svg]: What's this "in-darkBlue"? Give it a better more clear name if you don't mind 🙏

Comment (filipebfernandes) [reply src/assets/images/in-darkBlue.svg]: t's the linkedin icon xD I'll change it, thanks

File: src/components/PartnerCardMobile/index.css (hunk 1 above)
Comment (filipebfernandes) [reply src/components/PartnerCardMobile/index.css]: 👍

**Comment by jfranciscosousa** [comment]
@filipebfernandes rebase with master please, Netlify previews only work if the base is the `master` branch

**Pull request merged**